source/index.ts

```diff
@@ -136,8 +136,8 @@ namespace is { // tslint:disable-line:no-namespace
 	// tslint:enable:variable-name
 
 	export const nullOrUndefined = (value: any): value is null | undefined => null_(value) || undefined(value);
-	export const numericString = (value: any): value is String =>
-		!Number.isNaN(Number(value)) && !nullOrUndefined(value);
+	export const numericString = (value: string): bool =>
+		!Number.isNaN(Number(value));
```

Comment:
why did you remove the `nullOrUndefined` check?

Reply:
I removed it seeing that if `value` is either `null` or `undefined`, the TS compiler would immediately return an error (since the parameter is not a `string`).
In case we revert to accepting `any`, the `nullOrUndefined` check would be required again.

Reply:
Got it: add a `is.string` check. and then you can change `value` back to being `any`.

```ts
export const numericString = (value: any) =>
    is.string(value) && !Number.isNaN(Number(value));
```

Reply:
I believe the intent here is to check if a variable contains any representation of a number:

```js
if (is.number(val) || is.numericString(val)) {
   console.log(`The unicorn is ${val}`)
}
```

With that I mind, I it should accept `any` and type check `string` internally.
```

Reply:
Yes, it should accept `any`, like all the other checks.